### PR TITLE
[Translation][spanish] add spanish config.toml

### PR DESCRIPTION
**Pull request by mateolafalce**

### CODE REVIEW

File: blog/config.toml [hunk 1]
```diff
@@ -175,3 +175,21 @@ translated_content_notice = "이것은 커뮤니티 멤버가 <strong><a href=\"
 translated_by = "번역한 사람 : "
 translation_contributors = "With contributions from"
 word_separator = "와"
+
+# Spanish
+[languages.es]
+title = "Writing an OS in Rust"
+description = "This blog series creates a small operating system in the Rust programming language. Each post is a small tutorial and includes all needed code."
```
Comment (phil-opp) [inline blog/config.toml:182]: Feel free to translate the description too, if you like!

**Review by phil-opp — APPROVED** [review]
Thanks!

File: blog/config.toml (hunk 1 above)
Comment (mateolafalce) [reply blog/config.toml:182]: ok

**Comment by phil-opp** [comment]
@mateolafalce Should we merge this as-is or should we wait until the first post is translated?

**Pull request closed without merging**